e2e/libs/keywords/metrics_keywords.py

```
@@ -0,0 +1,49 @@
+        total_memory_in_mi = self.get_node_total_memory_in_mi(node_name)
+        memory_usage_in_percentage = memory_usage_in_mi / total_memory_in_mi * 100
+        logging(f'Got node {node_name} memory usage: {memory_usage_in_percentage} %')
+        return memory_usage_in_percentage
+
+    def check_if_node_under_memory_pressure(self, node_name):
+        logging(f"Checking if node {node_name} is under memory pressure")
+        condition_status = self.node.get_node_condition(node_name, "MemoryPressure")
+        if condition_status == "True":
+            logging(f"Node {node_name} is under memory pressure")
+            time.sleep(self.retry_count)
+            assert False, f"Node {node_name} is under memory pressure"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)